pages/docs/concepts/asyncapi-document/adding-operations.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Adding Operations
+weight: 60
+---
+
+In a messaging system, the term "operations" refers to the various methods by which messages are exchanged between participants or components. 
+
+## Features
+
+- Operations describe the behaviors and capabilities of the messaging channels described in the AsyncAPI document.
+
+- In a messaging channel, an operation represents a particular action or interaction that can be performed. 
+
+- The purpose of these operations is to provide a standardized means for describing the process of publishing, subscribing to, requesting, or replying to messages within the messaging system.
+
+## Adding Operations
+
+`operations` are no longer under `channels` in AsyncAPI Spec 3.0.0, instead, `operations` are separate objects in the AsyncAPI document on the same level. 
+`channels` can be linked within `operations` by referencing them within the `channels`, just like the following example -
+For adding operations to an AsyncAPI document, we need to define them within the channels section of the document. You can add operations to an AsyncAPI document as follows - 
+
+```
+onUserSignUp:
+  title: User sign up
+  summary: Action to sign a user up.
+  description: A longer description
+  channel:
+    $ref: '#/channels/userSignup'
+```
+
+Operations can be defined as an independent object in the AsyncAPI document. Operations have the following components for it's definition -
+- Locate the `channels` section in your AsyncAPI document. The `channels` section defines the messaging channels and their associated operations.
+
+|  Field Name | Type | Description |
+|---|---|---|
+| title | string | An easy to understand headline about the operation |
+| summary | string | A brief overview of the purpose of the operation |
+| description | string | A detailed explanation of the operation |
+| Channel | Reference Object Link | A `ref` pointer to the definition of the channel in which the operation is performed |
+| Action | "send" or "receive" | Uses `send` when the application sends a message to the given channel, and uses `receive` when the application receives a message from the given channel |
+| Tags | Tag Object | List of tags for logical grouping and categorization of operations |
+| Bindings | Bindings Object | A map where keys store the name of protocol and the values store protocol-specific definitions for the operation |
+| Traits | Traits Object | A list of traits to apply to the operation object. Traits must be merged using Traits Merge Mechanism. The resulting object should be a valid Operation Object |
+
+```
+onUserSignUp:
+  title: User sign up
+  summary: Action to sign a user up.
+  description: A longer description
+  channel:
+    $ref: '#/channels/userSignup'
+  action: send
+  tags:
+    - name: user
+    - name: signup
+    - name: register
+  bindings:
+    amqp:
+      ack: false
+  traits:
+    - $ref: '#/components/operationTraits/kafka'
+```
+
+## Types
```

Comment:
what is the purpose of this section and where did you get this content from? in v3 there is no longer publishing and subscribing - and if it comes to request/reply, there will be separate concept document that you should just link to

maybe instead of Types you mean actions? `send` and `receive` where `send` means that application describe in AsyncAPI document "sends" a message and `receive` means it is "receiving" a message

Reply:
since this section is no more relevant to v3 I have removed that section. `send` and `receive` is already defined in the table of spec3 doc - https://v3.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.12#operationsObject
which I already linkedin in the beginning of the document